src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -193,6 +195,25 @@ public MonitoredTrip(OtpDispatcherResponse otpDispatcherResponse) throws URISynt
         initializeFromItineraryAndQueryParams();
     }
 
+    /**
+     * Checks that, for each query provided, an itinerary exists.
+     * @return a summary of the itinerary existence results for each day of the week
+     */
+    public boolean checkItineraryExistence(
+        boolean replaceItinerary,
+        Function<OtpRequest, OtpResponse> otpResponseProvider
+    ) throws URISyntaxException {
+        // Get queries to execute by date.
+        List<OtpRequest> queriesByDate = getItineraryExistenceQueries();
+        this.itineraryExistence = new ItineraryExistence(queriesByDate, this.itinerary, this.arriveBy, otpResponseProvider);
+        this.itineraryExistence.checkExistence(this);
+        boolean itineraryExists = this.itineraryExistence.allMonitoredDaysAreValid(this);
+        // If itinerary should be replaced, do so if all checked days are valid.
+        return replaceItinerary && itineraryExists
```

Comment:
The return value is never used. Also, given that the method is only used in `ItineraryUtilsTest`, should it be moved there instead?

Reply:
This was an unfinished refactor fixed in https://github.com/ibi-group/otp-middleware/pull/240/commits/df3b4b62278435205497ff8c28f88e9169faf815... Thanks for catching it!